frontend/www/js/omegaup/components/common/Requests.vue

```diff
@@ -11,26 +11,24 @@
           <th>{{ T.requestDate }}</th>
           <th>{{ T.currentStatus }}</th>
           <th>{{ T.lastUpdate }}</th>
-          <th>{{ T.contestAdduserAddContestant }}</th>
+          <th>{{ textAddParticipant }}</th>
         </tr>
       </thead>
       <tbody>
         <tr v-for="request in requests">
           <td>{{ request.username }}</td>
           <td>{{ request.country }}</td>
           <td>{{ request.request_time }}</td>
-          <td v-if="request.last_update == null">{{ T.wordsPending }}</td>
-          <td v-else-if="request.accepted == 'true' || request.accepted == '1'">
+          <td v-if="request.last_update === null">{{ T.wordsPending }}</td>
+          <td v-else-if="request.accepted === true">
```

Comment:
```suggestion
          <td v-else-if="request.accepted">
```

como `request.accepted` ya es un booleano de verdad, el `=== true` ya no es necesario! igual en L31, se puede usar `!request.accepted`.

Reply:
Listo